lib/src/dart.dart

```diff
@@ -102,21 +101,22 @@ Future compile(String entrypoint, CompilerProvider provider,
   if (environment == null) environment = {};
   if (commandLineOptions != null) options.addAll(commandLineOptions);
 
-  if (packageRoot == null) {
-    packageRoot = p.join(p.dirname(entrypoint), 'packages');
+  if (packageConfig == null) {
+    packageConfig = p.join(p.dirname(entrypoint), '.packages');
   } else {
-    packageRoot = p.normalize(p.absolute(packageRoot));
+    packageConfig = p.normalize(p.absolute(packageConfig));
   }
 
   await compiler.compile(
       p.toUri(entrypoint),
       provider.libraryRoot,
-      p.toUri(appendSlash(packageRoot)),
```

Comment:
Are we still using `appendSlash()`? If not, can you delete it?

Reply:
Done.